pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -224,15 +225,125 @@ func ExamplePresentation_SetCredentials() {
 		},
 	}
 
+	vcStr := `
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
```

Comment:
is there a reason for this particular example? this is no longer valid without the context.

Reply:
This is what we have now in `verifiable` package. A majority of examples use `https://www.w3.org/2018/credentials/examples/v1` context and `UniversityDegreeCredential` type. But if we make JSON-LD compaction, we will get the document without the `UniversityDegreeCredential` type and any other VC data model extension fields. So what can we do in this regards:
1) provide only examples with base `https://www.w3.org/2018/credentials/v1` context.
2) develop our own data model extension in the form of a custom JSON-LD context and start a server which supplies this schema whenever the VC test runs.

What do you think @troyronda ?

Reply:
Just in case I changed this example to the basic one from VC Test Suite (with `AlumniCredential` type).